assets/javascripts/discourse/components/chat-live-pane.js

```diff
@@ -804,7 +808,6 @@ export default Component.extend({
 
   @action
   composerValueChanged(composerValue) {
-    this.reStickScrollIfNeeded();
```

Comment:
It looks to me like this recalculation is unnecessary on every keystroke. With the workaround above, it also causes lots of jitteriness...

Reply:
The composer height can change on composer value change, and if the composer height changes we might need to scroll..

Reply:
I'm sure I am missing context, but why do we need to scroll?

Reply:
previously, when the composer changed height it would cause the chat-live-pane to scroll... can't replicate right now if I comment out the line `this.reStickScrollIfNeeded();`, so maybe we are good to remove now